tests/nested/manual/minimal-smoke/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: execute smoke tests in a nested Ubuntu Core VM that meets the minimal requirements
+
+systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-20.04-64]
```

Comment:
I would add an extra worker for those systems in google-nested backend. These manual tests take too much time ans as we still run the nightly suite in travis we have the 50 minutes time limit.

Reply:
Added